coffeebot

```diff
@@ -1,3 +1,3 @@
 # /bin/bash
-./gradlew --build-cache installDist
+./gradlew --build-cache installDist &&
```

Comment:
does this still run or do we need to indicate continued line ```\``` ?

Reply:
No, && acts as a line continuer in bash!